client/lib/signup/step-actions/index.js

```diff
@@ -191,7 +192,11 @@ function getNewSiteParams( {
 		validate: false,
 	};
 
-	newSiteParams.options.wpcom_coming_soon = getNewSiteComingSoonSetting( state );
+	if ( config.isEnabled( 'coming-soon-v2' ) ) {
```

Comment:
We're using `'editing-toolkit/coming-soon'` over in https://github.com/Automattic/wp-calypso/pull/45892/files#diff-08c61c7c946b2c05d7687b49ef35985bR108

Pulled from the original https://github.com/Automattic/wp-calypso/pull/45606/files#diff-4663f7e9182cfd61c9e36163548cfdddR369

I don't have a strong opinion on the flag name. Just noting it so we can update in the appropriate places after we settle on which term we use.

Reply:
I've updated my branch to use `coming-soon-v2` in https://github.com/Automattic/wp-calypso/pull/45892/commits/e927805008cc3a55c64e05c13150044839e567eb